argh_derive/src/lib.rs

```diff
@@ -389,6 +410,13 @@ fn impl_from_args_struct(
                     });
                 }
 
+                if __dump_args {
+                    return std::result::Result::Err(argh::EarlyExit {
+                        output: __args_dump.join(" ").to_string(),
```

Comment:
I don't know why anyone would do it, but while argh rejects options that have spaces in them, it doesn't have a check for subcommands with spaces in them:

```
use argh::FromArgs;

#[derive(FromArgs)]
#[argh(description="")]
struct Cmd {
    #[argh(subcommand)]
    sub: SubCmd,
}

#[derive(FromArgs)]
#[argh(subcommand, name = "sub sub", description="")]
struct SubCmd {}

fn main() {
    let _cmd: Cmd = argh::from_env();
}
```

So technically we might need to escape the output, but I think it's fair to say this is a bug, and we shouldn't allow subcommands to contain whitespace. I filed https://github.com/google/argh/issues/83 to track that.

Reply:
Ack. 

Should we add an escape here for the time being

Reply:
Went ahead and escaped subcommands with spaces in the names.